jjb/anax/anax.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: jenkins-job
```

Comment:
This needs to be less generic, e.g. `anax-verify-pipeline` or something that similarly describes the project and function.